src/main/java/org/wildfly/security/http/HttpConstants.java

```diff
@@ -45,6 +45,7 @@ private HttpConstants() {
     private static final String CONFIG_BASE = HttpConstants.class.getPackage().getName();
     public static final String CONFIG_CONTEXT_PATH = CONFIG_BASE + ".context-path";
     public static final String CONFIG_REALM = CONFIG_BASE + ".realm";
+    public static final String CONFIG_ALGORITHM = CONFIG_BASE + ".algorithm";
```

Comment:
Is this used anywhere?  I think this was part of a prior version of this PR before it evolved further?